bogenliga/src/app/app.routing.ts

```diff
@@ -6,6 +6,7 @@ export const ROUTES: Routes = [
   {path: '', redirectTo: 'home', pathMatch: 'full'},
   {path: 'home', component: HomeComponent},
   {path: 'wettkaempfe', loadChildren: 'src/app/modules/wettkampf/wettkampf.module#WettkampfModule'},
+  {path: 'wettkaempfe/:id', loadChildren: 'src/app/modules/wettkampf/wettkampf.module#WettkampfModule'},
```

Comment:
/:id ist doch im WettkampfModule bei den Routen schon hinterlegt? Das könnt ihr hier also raus nehmen.

Reply:
Wenn ich das hier rausnehme bekomme ich den Link nicht mehr zum laufen. Ich hatte das in den Verwaltungsmodulen auch so gesehen, dachte aber, da dort die Verlinkung ja innerhalb des Moduls stattfindet, dass dies desweegen dort so funktioniert. Unser Link komt ja von Ligatabelle zu Wettkamp, also über zwei Module. Oder habe ich da einen Denkfehler?